### DISCUSSION TITLE
Question about the features

### DISCUSSION THREAD

**Discussion by Stick424** [Mr-MIBonk/M.I.B._More-Incredible-Bash#142] [General]
Hello everyone,

I have installed MIB and everything is working. However, when I read the M.I.B-MENU-STRUCTURE page, there are some features that catch my attention but I don't really know what they are for, for example the "skin screen" ? or the "select navigation to activate" ? 

Also, are there any nice features to install? I have for the moment the dual navigation installed. Is it possible to change the VC skin for example? 

Thanks for your answers!

**Comment by harman-f** [comment]
Which FW are you on again?

**Reply by Stick424** [reply]
Hello Harman, its MHI2Q_ER_AUG22_P5092 1329

**Reply by harman-f** [reply]
You could check out:

- Welcome sound 
- in case of internal amp -> subwoofer knob
- Dual Navigation with navignore
- VIM
- Some coding for audio ripping from CD
- …

Difficult to list all small things 😉
I might even have forgotten about some already 🤣
